runtime/onert/core/src/compiler/Compiler.cc

```diff
@@ -106,22 +91,12 @@ std::shared_ptr<CompilerArtifact> Compiler::compile(void)
   // TODO: Support hybrid execution -
   //       execution between interpreter and compiled executor (including control flow)
   if (_voptions[0]->disable_compile)
-  {
-    if (model_count > 1)
-      throw std::runtime_error{"NYI: Disable compilation for multi model is not supported yet"};
-
-    auto executors = std::make_shared<exec::Executors>();
-
-    _nnpkg->primary_model()->iterate([&](const ir::SubgraphIndex &index, ir::Graph &subg) {
-      executors->emplace(ir::ModelIndex{0}, index, std::make_unique<interp::InterpExecutor>(subg));
-    });
-    return std::make_shared<CompilerArtifact>(executors, nullptr);
-  }
+    throw std::runtime_error{"NYI: Disable compilation for multi model is not supported yet"};
```

Comment:
@hseok-oh Is it enough to check `_voptions[0]` only? If not, we may introduce option iteration loop to check `disable_compile` and `he_profiling_mode` in the loop.

Reply:
I'll move option check into for loop.